src/customizations/volto/components/manage/Blocks/Image/style.less

```diff
@@ -1,12 +1,126 @@
 .image-block {
   position: relative;
+  width: unset !important;
+
+  .edit {
+    min-height: 70px;
+  }
+
+  .large {
+    min-height: 500px;
+  }
+
+  .medium {
+    min-height: 250px;
+  }
+
+  .small {
+    min-height: 100px;
+  }
+}
+
+.block.align.right .imageElement {
+  margin-right: 0;
+  margin-left: 1em;
+  float: right;
+}
+
+.block.align {
+  .medium {
+    width: 50%;
+  }
+
+  .small {
+    width: 25%;
+  }
+}
+
+.block.align.right .large {
+  width: 50%;
+}
+
+.block.align.right .medium {
+  width: 25%;
+}
+
+.block.align.right .small {
+  width: 10%;
+}
+
+.block.align.left .imageElement {
+  margin-right: 1em;
+  margin-left: 0;
+  float: left;
+}
+
+.block.align.left .large {
+  width: 50%;
+}
+
+.block.align.left .medium {
+  width: 25%;
 }
 
-.copyright-image {
+.block.align.left .small {
+  width: 10%;
+}
+
+.image-block.full {
+  .eea.copyright {
+    position: absolute;
+    bottom: 15px;
+    width: 100%;
+    z-index: 5;
+  }
+}
+
+.block.image.align.full .image-block .copyright-image {
+  top: 95%;
+}
+
+.copyright-image-block {
   position: absolute;
-  z-index: 1000;
   top: 90%;
   width: 100%;
-  padding-right: 10px;
-  padding-left: 10px;
+  height: 25px;
+  padding: 0em 1em;
+  margin: 1rem 0em;
+}
+
+#copyright-hovering-text-inactive-right {
+  display: none;
+}
+
+#copyright-hovering-text-inactive-left {
+  display: none;
+}
+
+#copyright-hovering-text-active-right {
```

Comment:
@andreiggr  don't use id's .. there will be more than one image
with copyright info thus you would get duplicate id's